### PR TITLE
Documentation left pane view cuts title name #39621

### PR DESCRIPTION
**Pull request by KhushPatibandha**
resolved issue #39621 
image

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
Sorry, but I think this is the wrong fix.

Looking at the [deploy-preview-40275](https://deploy-preview-40275--kubernetes-io-main-staging.netlify.app/), there are few pages I spotted where the overall website pages look broken.

Eg: https://deploy-preview-40275--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

Attaching screenshot below (default zoom on MacOS default resolution):

**_Deploy Preview Page (**with PR changes**)_**
deploy-preivew


**_Current [Page](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) at present (without PR changes)_**
without-change


**_Full webpage screenshot of deploy preview_**
1-deploy-preview-40275

**Comment by KhushPatibandha** [comment]
> Sorry, but I think this is the wrong fix.
> 
> Looking at the [deploy-preview-40275](https://deploy-preview-40275--kubernetes-io-main-staging.netlify.app/), there are few pages I spotted where the overall website pages look broken.
> 
> Eg: https://deploy-preview-40275--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
> 
> Attaching screenshot below: **_Top content_** 1-deploy-preview-40275
> 
> **_Bottom content_** 2-deploy-preview-40275

May I ask what screen size you are on?

**Comment by dipesh-rawat** [comment]
> May I ask what screen size you are on?

@KhushPatibandha  I'm on native default MacOS resolution of 1728 x 1117. I have update the previous comment's screenshot. The current [page](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) without the PR changes looks fine without the issue highlighted in the screenshot.

File: layouts/docs/baseof.html [hunk 1]
```diff
@@ -17,10 +17,10 @@
     <div class="container-fluid td-outer">
       <div class="td-main">
         <div class="row flex-md-nowrap">
-          <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
+          <div class="col- col- col- td-sidebar d-print-none">
```
Comment (dipesh-rawat) [inline layouts/docs/baseof.html:20]: IMO the removal of the Bootstrap CSS class is causing the site grid layout to break in the deploy preview.

Refer: [Bootstrap Grid system doc](https://getbootstrap.com/docs/4.0/layout/grid/)

**Comment by KhushPatibandha** [comment]
Sure I'll look into it today and get back to you asap.

**Comment by KhushPatibandha** [comment]
I still am stuck on this problem and can't really figure out what's going wrong but I've thought of another approach that I can try 
image
Now the problem here is that the the sidebar is resizable but the main documentation is not, so basically the sidebar is resizing but the main documentation is not shrinking. can you help me with that ?

**Comment by tengqm** [comment]
I believe you will have to introduce some jQuery component to add a vertical splitter or things like that.

**Comment by KhushPatibandha** [comment]
image
@dipesh-rawat How about this ?

**Comment by KhushPatibandha** [comment]
> image
> 
> @dipesh-rawat How about this ?

IF this look ok they can you please help me locate the css files for this
image

**Comment by KhushPatibandha** [comment]
> @KhushPatibandha You may want to check #40369 and see if you can continue working towards that direction. I'm too occupied to fine tune the css look and feel.

I'll surely have a look.

**Comment by divya-mohan0209** [comment]
Hello @KhushPatibandha , gentle reminder! Are you still working on this? If not, it is completely okay to say so since we understand that you may have bandwidth issues & can't continue working.

In any case, please let us know by 15th August, 2023. If there is no response from your side, we shall be closing off the PR.

/remove-lifecycle stale

**Comment by divya-mohan0209** [comment]
Closing the PR off per above comments.
/close

**Pull request closed without merging**